src/utils/http.ts

```diff
@@ -8,6 +8,18 @@ export function setBaseUrl(url: string) {
     baseURL: url,
     timeout: 15000 // timeout so we're not getting ECONNRESET
   });
+
+  // set teh tracker version and ID
+  let version = process.env.npm_package_version;
```

Comment:
env variables should be all caps `NPM_PACKAGE_VERSION` and `NPM_PACKAGE_NAME`

Reply:
my logging showed lowercase for these. i can add a test

Reply:
oh interesting. i thought these were set by us and not by the package automatically

Reply:
since this is a library and is running the vscode environment, this doesn't really make sense because `npm_package_version` or `npm_package_name` could refer to any package in vscode

Reply:
so i think we should just have these come from the package.json. Doesn't really make sense to be an environment variable. right?

Reply:
thats an interesting point. i think you're right. i'll look up the source of what gave me that answer but yes, makes sense to just use the package.json. my latest commit tested both cases but i'll just update to use the packge.json. thanks.